linode-launch.py

```diff
@@ -136,21 +188,34 @@ def create(*args, **kwargs):
         os._exit(1)
 
 def launch(client):
-    datacenters = client.avail_datacenters()
-    plans = client.avail_linodeplans()
-    distributions = client.avail_distributions()
-    kernels = client.avail_kernels()
-
-    datacenter = filter(lambda d: d[u'LOCATION'].lower().find(CLUSTER['datacenter'].lower()) >= 0, datacenters)[0][u'DATACENTERID']
-    distribution = filter(lambda d: d[u'LABEL'].lower().find(CLUSTER['distribution'].lower()) >= 0, distributions)[0][u'DISTRIBUTIONID']
```

Comment:
why expand these into loops? doesn't feel relevant to the stated goal of this pr